src/__test__/components/Header.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { fireEvent, render } from '@testing-library/react'
+import Header from '../../components/Header'
+
+describe('Header testing', () => {
+	it('Renders without crashing', () => {
```

Comment:
You can delete this test, since you are just triggering a render, but not proving any behavior or content in the component.